client/app/components/QueryEditor.jsx

```diff
@@ -102,9 +102,11 @@ class QueryEditor extends React.Component {
     this.onLoad = (editor) => {
       // Release Cmd/Ctrl+L to the browser
       editor.commands.bindKey('Cmd+L', null);
-      editor.commands.bindKey('Ctrl+P', null);
       editor.commands.bindKey('Ctrl+L', null);
 
+      // golineup only mac
+      editor.commands.bindKey({ win: null, mac: 'Ctrl+P' }, 'golineup');
```

Comment:
I think you still need to unbind it for Windows, to allow for our shortcut to take over.

Reply:
I was thinking, this line becomes unnecessary once you unbind the key only for Windows, since golineup is the [default behavior](https://github.com/ajaxorg/ace/wiki/Default-Keyboard-Shortcuts) for mac :thinking:

Reply:
I think so, too. But it didn't work when I try.

Reply:
pretty weird though :open_mouth:... Anyway, the parameter dialog is working with Ctrl+P in my Linux :+1: